src/frontend/layouts/nav-top-bar/nav-top-bar.tsx

```diff
@@ -29,6 +42,7 @@ const NavTopBar: React.FC = () => {
       <Navbar.Toggle aria-controls="nav-top-bar-items" />
       <Navbar.Collapse id="nav-top-bar-items">
         <Nav className="ml-auto">
+          <Button onClick={swap}>{themeIndex === 0 ? 'Light' : 'Night'}</Button>
```

Comment:
I was actually hoping to have this be located on the settings page, but I'm open to reasons why not if you think it's better elsewhere.

Also, I think you probably could do this as a dropdown and then you could avoid hard-coding the label text and have `themes.map(t => t.themeName)` or something

Reply:
i think as of right now i want it on the main page because of the whole refresh cancels your selection thing so I don't want users to have to go back to settings every time they want to change it back. once we set it up as being part of the settings so that it's persistent and stuff I agree